From 3be2f9c247ba84f98c7d7864fa826aa67162ec4c Mon Sep 17 00:00:00 2001
From: Anson Huang <b20788@freescale.com>
Date: Mon, 13 Feb 2012 12:02:14 +0800
Subject: [PATCH 276/442] ENGR00174306 Update autorun-rtc script for 3.0.15

Previous script check whether the kernel version
is after 38, but now we use 3.0.15, which will cause
the script recognize the kernel as before 2.6.38 by mistake,
we should remove this checking, as we will only support
2.6.38 and the following kernel version.

Signed-off-by: Anson Huang <b20788@freescale.com>
---
 test/mxc_rtc/autorun-rtc.sh | 7 -------
 1 file changed, 7 deletions(-)

diff --git a/test/mxc_rtc/autorun-rtc.sh b/test/mxc_rtc/autorun-rtc.sh
index 219fc94..4b33658 100755
--- a/test/mxc_rtc/autorun-rtc.sh
+++ b/test/mxc_rtc/autorun-rtc.sh
@@ -11,14 +11,7 @@ echo "$(platform)"
 if [ "$(platform)" != IMX31ADS ] && [ "$(platform)" != IMX32ADS ] \
     && [ "$(platform)" != IMX25_3STACK ] && [ "$(platform)" != IMX35_3STACK ]; then
 	rtc_test_param=--no-periodic
-	# For kernel version 2.6.38 and higher, number of interrupts expected will be 11
-	VER=`echo $(kernel_version) | cut -d. -f3 | cut -d- -f1`
-	REF_VER=38
-	if [ $VER -ge $REF_VER ]; then
 		RTC_IRQS_EXPECTED=11
-	else
-		RTC_IRQS_EXPECTED=1
-	fi
 else
 	rtc_test_param=--full
 	RTC_IRQS_EXPECTED=131
-- 
1.8.0

